docker/Dockerfile

```diff
@@ -76,8 +78,16 @@ RUN mkdir -p /app && \
 USER app
 WORKDIR /app/devito
 
-ENV DEVITO_ARCH="gcc"
-ENV DEVITO_LANGUAGE="openmp"
+# Env vars from ARGS
+ARG arch="gcc"
+ARG language="openmp"
+ARG logging="DEBUG"
```

Comment:
I don't think we want this, and if there was already in master, we should maybe drop it?

Reply:
Sure. INFO?